panels/dev-info/ha-panel-dev-info.html

```diff
@@ -86,6 +86,23 @@
             <a href='https://github.com/balloob/home-assistant-js' target='_blank'>frontend-core</a>
           </p>
           <p>
+            Documentation: 
+            <a href='https://home-assistant.io/' target='_blank'>Home Page</a> &mdash;
+            <a href='https://home-assistant.io/components/' target='_blank'>Components</a> &mdash;
+            <a href='https://home-assistant.io/cookbook/' target='_blank'>Examples</a><br />
+            Help: 
+            <a href='https://home-assistant.io/getting-started/' target='_blank'>Getting Started</a> &mdash;
+            <a href='https://community.home-assistant.io/' target='_blank'>Forum</a> &mdash;
+            <a href='https://gitter.im/home-assistant/home-assistant' target='_blank'>Gitter Chatroom</a> &mdash;
+            <a href='https://twitter.com/home_assistant' target='_blank'>Twitter</a> &mdash;
+            <a href='https://reddit.com/r/homeassistant' target='_blank'>Reddit</a> &mdash;
+            <a href='https://plus.google.com/u/0/b/110560654828510104551/communities/106562234893511202708' target='_blank'>Google+</a><br />
+            Have you found an issue in your Home Assistant installation? Please report it! 
+            <a href='https://github.com/home-assistant/home-assistant/issues' target='_blank'>Issue tracker Home Assistant</a> &mdash;
+            <a href='https://github.com/home-assistant/home-assistant.io/issues' target='_blank'>Issue tracker home-assistant.io (website and documentation)</a> &mdash;
+            <a href='https://community.home-assistant.io/c/feature-requests' target='_blank'>Feature requests for Home Assistant</a>
+          </p>             
+          <p>
```

Comment:
Putting all these links into the project directly makes it difficult to maintain if something was to change. There should only be a link to https://home-assistant.io/help/ to keep things simple. I suggest that the PR be closed.